### PR TITLE
[QuartzCore] Improve headers that appear generated

### PR DESCRIPTION
**Pull request by leptos-null**
I think `QuartzCore/QuartzCore-Structs.h` should eventually be removed, or a `#warning` added to the file. I didn't make a decision about this due to backwards-compatibility concerns.

### CODE REVIEW

File: QuartzCore/CAFilter.h [hunk 1]
```diff
@@ -1,58 +1,156 @@
...
 @property (getter=isEnabled) BOOL enabled;
-@property BOOL cachesInputImage;
-// ["multiplyColor","multiplyGradient","gaussianBlur","pageCurl","fog","lighting","clear","copy",
-//  "sourceOver","sourceIn","sourceOut","sourceAtop","destOver","destIn","destOut","destAtop","xor","plusL","multiply"]
+@property BOOL cachesInputImage API_AVAILABLE(ios(3.0));
+
 + (NSArray <NSString *> *)filterTypes;
 + (CAFilter *)filterWithType:(NSString *)type;
-+ (CAFilter *)filterWithName:(NSString *)name;
++ (CAFilter *)filterWithName:(NSString *)name; // alias for filterWithName:
```
Comment (PoomSmart) [inline QuartzCore/CAFilter.h:14]: alias for filterWithType: ?

File: QuartzCore/CAFilter.h [hunk 2]
```diff
@@ -1,58 +1,156 @@
...
-- (Object *)CA_copyRenderValue;
-- (void)willChangeValueForKey:(id)key;
-- (void)didChangeValueForKey:(id)key;
 
 @end
 
+// only some of the below strings are used for
+// filter types.
+// it's best to use +[CAFilter filterTypes] to
+// detemine the available filters.
```
Comment (PoomSmart) [inline QuartzCore/CAFilter.h:27]: determine *

File: QuartzCore/CAFilter.h (hunk 1 above)
Comment (leptos-null) [reply QuartzCore/CAFilter.h:14]: yes, that makes more sense. thanks (just double-checked in Hopper as well)

File: QuartzCore/CAFilter.h (hunk 2 above)
Comment (leptos-null) [reply QuartzCore/CAFilter.h:27]: Thanks. I'll push the fixes

**Review by PoomSmart — APPROVED** [review]

**Pull request merged**